contracts/bprotocol/Registry.sol

```diff
@@ -117,10 +119,19 @@ contract Registry is Ownable {
         address _avatar = address(new Avatar(bComptroller, comptroller, comp, cEther, address(this)));
         avatarOf[_owner] = _avatar;
         ownerOf[_avatar] = _owner;
+        avatars.push(_avatar);
         emit NewAvatar(_avatar, _owner);
         return _avatar;
     }
 
+    function avatarSize() external view returns (uint256) {
+        return avatars.length;
+    }
+
+    function getAvatarAt(uint256 index) external view returns (address) {
```

Comment:
do we need it? give array is public?
if it is best practice now, maybe make the array internal.
But i think it is best to just remove this function.